src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -436,10 +436,13 @@ uct_rc_mlx5_txqp_dptr_post(uct_rc_mlx5_iface_common_t *iface, int qp_type,
         break;
 
     case MLX5_OPCODE_RDMA_READ:
-        fm_ce_se |= uct_rc_mlx5_ep_fm(iface, txwq);
-        /* Fall through */
     case MLX5_OPCODE_RDMA_WRITE:
         /* Set RDMA segment */
+        fm_ce_se |= uct_rc_mlx5_ep_fm(iface, txwq,
```

Comment:
according to this , RDMA_WRITE will always have MLX5_WQE_CTRL_CQ_UPDATE, meaning CQ moderation is disabled - and this is not what we want